### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by EDDragonWolf**
Please. review but not merge

### CODE REVIEW

File: src/supernode/baseclientproxy.cpp [hunk 1]
```diff
@@ -114,48 +117,50 @@ bool supernode::BaseClientProxy::CreateAccount(const supernode::rpc_command::CRE
...
     out.Seed = seed;
-    out.Result = STATUS_OK;
     return true;
 }
 
-bool supernode::BaseClientProxy::RestoreAccount(const supernode::rpc_command::RESTORE_ACCOUNT::request &in, supernode::rpc_command::RESTORE_ACCOUNT::response &out)
+bool supernode::BaseClientProxy::RestoreAccount(const supernode::rpc_command::RESTORE_ACCOUNT::request &in, supernode::rpc_command::RESTORE_ACCOUNT::response &out, epee::json_rpc::error &er)
 {
-	LOG_PRINT_L0("BaseClientProxy::RestoreAccount" << in.Seed);
+    LOG_PRINT_L0("BaseClientProxy::RestoreAccount " << in.Seed);
```
Comment (mbg033) [inline src/supernode/baseclientproxy.cpp]: Might be a security issue (seed in the logs)

File: src/supernode/baseclientproxy.cpp [hunk 2]
```diff
@@ -114,48 +117,50 @@ bool supernode::BaseClientProxy::CreateAccount(const supernode::rpc_command::CRE
...
     out.Seed = seed;
-    out.Result = STATUS_OK;
     return true;
 }
 
-bool supernode::BaseClientProxy::GetSeed(const supernode::rpc_command::GET_SEED::request &in, supernode::rpc_command::GET_SEED::response &out)
+bool supernode::BaseClientProxy::GetSeed(const supernode::rpc_command::GET_SEED::request &in, supernode::rpc_command::GET_SEED::response &out, epee::json_rpc::error &er)
 {
-	LOG_PRINT_L0("BaseClientProxy::GetSeed" << in.Account);
+    LOG_PRINT_L0("BaseClientProxy::GetSeed " << in.Account);
```
Comment (mbg033) [inline src/supernode/baseclientproxy.cpp]: Is there any sensitive info in the `in.Account` ?

File: src/supernode/baseclientproxy.cpp (hunk 1 above)
Comment (EDDragonWolf) [reply src/supernode/baseclientproxy.cpp]: Removed seed

File: src/supernode/baseclientproxy.cpp (hunk 2 above)
Comment (EDDragonWolf) [reply src/supernode/baseclientproxy.cpp]: No, removed

**Comment by EDDragonWolf** [comment]
> Maybe make sense to reformat all the "supernode" code so indentation will be 4 spaces? Right now it mixed with tabs and hard to read while reviewing on github
Agree, fixed spaces and some other formatting

**Pull request closed without merging**